src/server/mod.rs

```diff
@@ -241,7 +245,11 @@ impl Server {
         let tokenserver_state = if settings.tokenserver.enabled {
             Some(tokenserver::ServerState::from_settings(
                 &settings.tokenserver,
-                metrics.clone(),
+                metrics::metrics_from_opts(
```

Comment:
Tokenserver will have its own separate metrics client -- is that okay? It seems like it shouldn't be a problem to me, but I wanted to double-check

Reply:
Yea, it's not a big deal for now.